internal/verifier/change_stream.go

```diff
@@ -318,7 +318,7 @@ func (csr *ChangeStreamReader) readAndHandleOneChangeEventBatch(
 	var curTs primitive.Timestamp
 	curTs, err := extractTimestampFromResumeToken(cs.ResumeToken())
 	if err == nil {
-		lagSecs := curTs.T - sess.OperationTime().T
+		lagSecs := int32(sess.OperationTime().T) - int32(curTs.T)
```

Comment:
This seems to be prone to overflow. Can we convert `T` (uint32) to int64?

Reply:
yes, good call